analyzer/general.py

```diff
@@ -0,0 +1,49 @@
+"""
```

Comment:
We can move these code to `__init__.py`. I think it is not needed.

Reply:
OK